<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeometricOpOutputWizardPageForm</class>
 <widget class="QWizardPage" name="GeometricOpOutputWizardPageForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_10">
   <item row="0" column="0">
    <widget class="QWidget" name="m_widgetOutput" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <widget class="QGroupBox" name="m_operationGroupBox">
          <property name="minimumSize">
           <size>
            <width>275</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Operation</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QCheckBox" name="m_convexHullCheckBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Convex Hull</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="m_centroidCheckBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Centroid</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="m_mbrCheckBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Minimum Bounding Rectangle</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QCheckBox" name="m_areaCheckBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Area</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QCheckBox" name="m_lineCheckBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Line</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QCheckBox" name="m_perimeterCheckBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Perimeter</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QGroupBox" name="m_methodologyGroupBox">
            <property name="title">
             <string>Methodology</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QRadioButton" name="m_allObjectsRadioButton">
               <property name="text">
                <string>All Objects</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QRadioButton" name="m_simpleRadioButton">
               <property name="text">
                <string>Aggregated Objetcts</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QRadioButton" name="m_byAttributesRadioButton">
               <property name="text">
                <string>By Attributes</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QComboBox" name="m_attributesComboBox">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="m_outputGroupBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Output</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Repository:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLineEdit" name="m_repositoryLineEdit">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QToolButton" name="m_targetFileToolButton">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QToolButton" name="m_targetDatasourceToolButton">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>BD</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Layer Name:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLineEdit" name="m_newLayerNameLineEdit">
               <property name="readOnly">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
